src/core/src/op/paged_attention.cpp

```diff
@@ -14,123 +14,125 @@ PagedAttentionExtension::PagedAttentionExtension(const ov::OutputVector& args) :
+                          "Input `sliding_window` should be a scalar but it has rank ",
+                          get_input_partial_shape(10).rank().get_length(),
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(10).is_dynamic() || get_input_element_type(10) == element::i32,
+                          "Element type of `sliding_window` input should be i32, but it is ",
                           get_input_element_type(10),
-                          ", shape ",
-                          get_input_shape(10));
+                          ".");
 
-    // alibi_slopes: 1D float tensor
     NODE_VALIDATION_CHECK(this,
-                          // get_input_element_type(11) == ov::element::f32 &&
-                          get_input_partial_shape(11).rank().get_length() == 1,
-                          "alibi_slopes should be a 1D float tensor. ",
-                          "Got element type ",
+                          get_input_partial_shape(11).rank().is_dynamic() || get_input_partial_shape(11).rank().get_length() == 1,
+                          "Rank of `alibi_slopes` input should be 1, but it is ",
+                          get_input_partial_shape(11).rank().get_length(),
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(11).is_dynamic() || get_input_element_type(11).is_real(),
+                          "Element type of `alibi_slopes` input should be a floating type, but it is ",
                           get_input_element_type(11),
-                          ", shape ",
-                          get_input_partial_shape(11));
-
-    // sliding_window: int scalar
+                          ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_partial_shape(12).rank().is_dynamic() || get_input_partial_shape(12).rank().get_length() == 0,
+                          "Input `max_context_len` should be a scalar but it has rank ",
+                          get_input_partial_shape(12).rank().get_length(),
+                          ".");
     NODE_VALIDATION_CHECK(this,
-                          // get_input_element_type(12) == ov::element::i32 &&
-                          get_input_partial_shape(12).rank().get_length() == 0,
-                          "sliding_window argument should be an i32 scalar. ",
-                          "Got element type ",
+                          get_input_element_type(12).is_dynamic() || get_input_element_type(12) == element::i32,
+                          "Element type of `max_context_len` input should be i32, but it is ",
                           get_input_element_type(12),
-                          ", shape ",
-                          get_input_partial_shape(12));
+                          ".");
 
-    set_output_type(0, query_type, query_shape);
+    set_output_type(0, get_input_element_type(0), get_input_partial_shape(0));
```

Comment:
May we change the shape from [batch, head_number * head_size] to [batch, 1, head_number * head_size] which can save a reshape op in the graph?